### PR TITLE
Create rule manager client

### PR DESCRIPTION
**Pull request by tjsilver**
## What does this change?
Adds a React/Typescript client to the rule manager app.

## How to test

In the root of the branch, run `sbt "ruleManager/run"` and navigate to https://typerighter.local.dev-gutools.co.uk/ where you should see the message "Hello world! This is the rule manager client!".

## How can we measure success?
Users will be (eventually) able to manage Typerighter rules through a website rather than having to manually edit a spreadsheet.

### CODE REVIEW

**Comment by tjsilver** [comment]
[Management: create 'hello world' React frontend for service](https://trello.com/c/QpsEoAMW/519-management-create-hello-world-react-frontend-for-service)

File: checker/app/views/manager.scala.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@main(title = "Rule manager", pageName = Some("Rule manager"), breadcrumbsList = List("Manager")) {
```
Comment (jonathonherbert) [inline checker/app/views/manager.scala.html:1]: Is this used?

File: rule-manager/rule-manager-client/dist/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/*
```
Comment (jonathonherbert) [inline rule-manager/rule-manager-client/dist/main.js:1]: Presume this isn't necessary to store in VCS?

File: checker/app/views/manager.scala.html (hunk 1 above)
Comment (tjsilver) [reply checker/app/views/manager.scala.html:1]: No, don't think so - will remove

**Review by jonathonherbert — APPROVED** [review]
Tested locally, works as expected! Looks like the relevant parts of the build are created in prod, but we'll only really be able to test this once we've a deployable service.

**Pull request merged**